Core/src/Propagator/SympyStepper.cpp

```diff
@@ -164,6 +165,8 @@ Result<double> SympyStepper::stepImpl(
       break;
     }
 
+    ++state.statistics.nFailedSteps;
```

Comment:
Is this necessarily an error here? 
Ah, failed probably refers to the number of trials until the accurracy target was reached. 

Maybe the name could be more explicit about that?

Reply:
maybe `rejected`?